ViSearchSDK/ViSearchSDK/Classes/Request/ViRequestSerialization.swift

```diff
@@ -21,6 +21,13 @@ open class ViRequestSerialization {
         return authString
     }
     
+    
+    /// generate the url with query string and escape parameter properly
+    public func generateRequestUrl( baseUrl: String , apiEndPoint: String , searchParams:[String: Any]) -> String {
```

Comment:
should have another enum here e.g. PSSearchEndpoints or ViProductSearchEndpoints